src/Microsoft.Health.Fhir.SqlServer/Features/Storage/TvpRowGeneration/BulkTokenQuantityCompositeSearchParameterV1RowGenerator.cs

```diff
@@ -36,13 +37,13 @@ internal class BulkTokenQuantityCompositeSearchParameterV1RowGenerator : BulkCom
                 row = new BulkTokenQuantityCompositeSearchParamTableTypeV1Row(
```

Comment:
>row [](http://example.com/codeflow?start=16&length=3)

Do we actually guarantee distinct for these rows

Reply:
yes we do. I will send you the request I tested this with